src/datachain/lib/hf.py

```diff
@@ -0,0 +1,152 @@
+try:
+    from datasets import (
+        Array2D,
+        Array3D,
+        Array4D,
+        Array5D,
+        Audio,
+        ClassLabel,
+        Dataset,
+        DatasetDict,
+        Image,
+        IterableDataset,
+        IterableDatasetDict,
+        Sequence,
+        Value,
+        load_dataset,
+    )
+    from datasets.features.features import string_to_arrow
+    from datasets.features.image import image_to_bytes
+
+except ImportError as exc:
+    raise ImportError(
+        "Missing dependencies for huggingface datasets:\n"
+        "To install run:\n\n"
+        "  pip install 'datasets'\n"
```

Comment:
[C] Looks like all of the other error messages of this form use `pip install 'datachain[optional]'`